commandapi-core/src/main/java/dev/jorel/commandapi/AbstractCommandAPICommand.java

```diff
@@ -31,6 +31,7 @@
 import dev.jorel.commandapi.exceptions.GreedyArgumentException;
 import dev.jorel.commandapi.exceptions.MissingCommandExecutorException;
 import dev.jorel.commandapi.exceptions.OptionalArgumentException;
+import org.jetbrains.annotations.NotNull;
```

Comment:
As opposed to `javax.annotation.Nonnull`? This removes the dependency on jetbrains which is unnecessary an excessive given Java already has one built-in.

Reply:
Oh right, that's what it was called. Thanks, will add. (I knew there was a native one, but typed `@Not` before checking suggestion)